docs/meta.md

```diff
@@ -21,8 +15,9 @@ for a snap package. The following keys are mandatory:
 
 The following keys are optional:
 
-* `icon`: a SVG icon for the snap that is displayed in the store
-* `explicit-license-agreement`: set to `Y` if the user needs to accept a
+* `summary`: A short summary
+* `description`: A long description
```

Comment:
before the description was optional (and you got a placeholder if it wasn't there). Is it now required?